cs/src/io/unsafe/MemoryHelper.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.ConstrainedExecution;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bond.IO.Unsafe
+{
+    internal unsafe class MemoryHelper
+    {
+        [DllImport("msvcrt.dll", EntryPoint = "memcpy", CallingConvention = CallingConvention.Cdecl, CharSet = CharSet.Unicode, SetLastError = false)]
```

Comment:
Do you think it would be worth to have a test just for this function, hitting all the cases of buffer length? We already have a unit test project for internal functions.